src/test/java/org/kitteh/irc/client/library/implementation/KICLPropertiesTest.java

```diff
@@ -0,0 +1,12 @@
+package org.kitteh.irc.client.library.implementation;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KICLPropertiesTest {
+
+    @Test
+    public void testVersion() throws Exception {
```

Comment:
That's what I was talking about earlier on IRC. There's no way I can properly test it, is there?